README.md

```diff
@@ -21,31 +21,39 @@ Currently the docker-compose performance isn't very good. Thus, for local dev, i
 
 We need to install airflow.
 
-1. First, we should create virtual environment with pyenv:
-
-```
-    export PYTHON_VERSION=3.8.13
-    pyenv install $PYTHON_VERSION
-    pyenv global $PYTHON_VERSION
-    pyenv virtualenv $PYTHON_VERSION workflows
-    pyenv activate workflows
-```
-
-3. Set airflow home directory:
-
-```
-export AIRFLOW_HOME=/path/to/cloned/repo
-```
-
-4. Install all required dependencies for a project. We need 3 requirements files, when we're running the project locally. First file (requirements.py) has all additional dependencies required to run the tasks correctly (connecting to ftputil, boto3 and etc.), the second one installs requirememnts for testing, such as pytest, the third- (requirements_airflow.py) installs Airflow itself and constraints needed for it. The the third file is not needed when we're running the procject on Docker, because we're using Apache Airflow Docker image:
-   `pip install -r requirements.txt -r requirements-test.txt -r requirements-airflow.txt`
-5. Run Airflow. Airflow comes with the `standalone` command. This should not be used as it seems to use the SequentialExecutor by default. To have our local config being use, we must run all services together :
-
-```
-    airflow webserver
-    airflow triggerer
-    airflow scheduler
-```
+1. First, we need to install poetry:
+
+   ```
+   curl -sSL https://install.python-poetry.org | python3 -
+
+   ```
+
+2. Install pyenv, the instruction can be found [here](https://github.com/pyenv/pyenv#installation)
+3. Export correct python version, create and activate virtual environment by using pyenv:
+   ```
+   export PYTHON_VERSION = 3.8.13
+   pyenv global $(PYTHON_VERSION)
+   pyenv install ${PYTHON_VERSION}
+   pyenv virtualenv ${PYTHON_VERSION} workflows
```

Comment:
no need this because we are using poetry which comes with each own vm and we are modifying all the commands to use it `poetry run ...`